src/app/services/content-fetcher.service.spec.ts

```diff
@@ -26,133 +25,98 @@ describe('ContentFetcherService', () => {
 
   afterEach(() => {
     httpTestingController.verify();
-
   });
 
   // Tests creation of fetcher
   it('should be created', () => {
     expect(service).toBeTruthy();
-
   });
 
-  // Here begins testing of methods in .ts file
-
-  /**
-   * First, we test the getAllContent function to make sure it pings
-   * correctly. To do so, mocking is needed. First stipulate the fakeAsync() 
-   * function to generate an empty response that one will manually 
-   * populate. Using service.getAllContent().subscribe() will give a 
-   * mocked response. Lastly do a test as necessary to test if the method
-   * is indeed a "GET" method, and use req.flush(response) and tick() to 
-   * finalize mock.
-   */
+  //getAllContent()
   it('getAllContent is working', fakeAsync(() => {
-    let response = {};
-
-    service.getAllContent().subscribe(
-      (receivedResponse: any) => { },
-      (error: any) => { }
-    );
+    service.getAllContent().subscribe();
     const req = httpTestingController.expectOne(baseURL + '/content');
     expect(req.request.method).toEqual("GET");
-    req.flush(response);
     tick();
-
   }));
 
-  // Similar work is done using getContentById. Must be mocked
+  //Testing getContentByID(id)
   it('getContentByID is working', fakeAsync(() => {
-    let response = {};
-
-    service.getContentByID(1).subscribe(
-      (receivedResponse: any) => { },
-      (error: any) => { }
-    );
+    service.getContentByID(1).subscribe();
     const req = httpTestingController.expectOne(baseURL + '/content/1');
     expect(req.request.method).toEqual("GET");
-    req.flush(response);
     tick();
-
   }));
 
   // Testing createNewContent(content) function
   it('createNewContent is working', fakeAsync(() => {
-    let response = {};
-    let content: Content = new Content(null, null, null, null, null, null)
-
-    service.createNewContent(content).subscribe(
-      (receivedResponse: any) => { },
-      (error: any) => { }
-    );
+    service.createNewContent(new Content(null, null, null, null, null, null)).subscribe();
     const req = httpTestingController.expectOne(baseURL + '/content');
     expect(req.request.method).toEqual("POST");
-    req.flush(response);
     tick();
+  }));
 
+   // Testing addLinkToContent(content) function
```

Comment:
At least there is a comment here, but perhaps a little bit more detail would be helpful. The next group of associates who work on this might want to also know a bit about the methodology that you are using to test. This also applies to all testing related comments.